examples/quickstart/tutorial/hadoop/docker/Dockerfile

```diff
@@ -36,12 +36,22 @@ RUN ssh-keygen -q -N "" -t rsa -f /etc/ssh/ssh_host_rsa_key
 RUN ssh-keygen -q -N "" -t rsa -f /root/.ssh/id_rsa
 RUN cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys
 
-# zulu java 8
-COPY setup-zulu-repo.sh /root/setup-zulu-repo.sh
-RUN /root/setup-zulu-repo.sh
-RUN yum install -y zulu-8
+#
+# Pull Zulu OpenJDK binaries from official repository:
+#
+
+ARG ZULU_REPO_VER=1.0.0-1
+
+RUN rpm --import http://repos.azulsystems.com/RPM-GPG-KEY-azulsystems && \
```

Comment:
this seems cleaner. 
You might want to update examples/quickstart/tutorial/hadoop3/docker/Dockerfle as well ?

Reply:
Thanks for the point. I think updating the examples/quickstart/tutorial/hadoop3/docker/Dockerfle would be good. Will update the PR with the update